staging/src/k8s.io/apiserver/pkg/storage/tests/cacher_test.go

```diff
@@ -356,10 +356,7 @@ func TestTooLargeResourceVersionList(t *testing.T) {
 
 	result := &example.PodList{}
 	err = cacher.List(context.TODO(), "pods/ns", storage.ListOptions{ResourceVersion: listRV, Predicate: storage.Everything}, result)
-	if !errors.IsTimeout(err) {
```

Comment:
timeout error is no longer checked ?

Reply:
It's checked as part of IsTooLargeResourceVersion.

Reply:
that seems a bit strange to me... timeout reason and TooLargeResourceVersion cause seem orthogonal to me

Reply:
reverted